challenges-26-plus.yml

```
@@ -0,0 +1,100 @@
+frame_loading:
+  question: |
+    Our site has a sidebar that takes forever to load, so our project
+    manager wanted us to extract it into a lazy Turbo Frame.
+    
+    Now he wants us to show a loading indicator whenever we are
+    loading its contents! Here's the loading code:
+    
+    ```html
+    <turbo-frame id="app_sidebar">
+        {% include('templates/_busySidebar.html.twig') %}
```

```suggestion
        {{ include('templates/_busySidebar.html.twig') }}
```

`{% include` is a think - but it's the not preferred way to do it anymore.